realm/realm-library/src/main/cpp/objectserver_shared.hpp

```diff
@@ -81,6 +81,10 @@ class JniSession {
                             local_env, obj, "notifySessionError", "(ILjava/lang/String;)V");
                     local_env->CallVoidMethod(
                             obj, notify_error_handler, error_code.value(), local_env->NewStringUTF(message.c_str()));
+                    if (local_env->ExceptionCheck() == JNI_TRUE) {
+                        local_env->ExceptionDescribe();
```

Comment:
This doesn't seem to be necessary? Especially where does it output to for Android?